src/controllers/plots.controller.ts

```diff
@@ -0,0 +1,163 @@
+import { plotsDB } from '../models/db.index';
+import { RequestHandler } from 'express';
+import createHttpError from 'http-errors';
+
+export const getPlots: RequestHandler = async (req, res, errFunc) => {
+    try {
+        const plots = await plotsDB.findAll();
+        res.status(200).json(plots);
+    } catch (error) {
+        errFunc(error);
+    }
+};
+
+export const getPlot: RequestHandler = async (req, res, errFunc) => {
```

Comment:
lets package the reference node (if there is one, need to check if nodeid is null)